ui/components/src/TimeSeriesTooltip/TimeSeriesTooltip.tsx

```diff
@@ -11,78 +11,73 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Box, Portal, Typography, Stack, Switch } from '@mui/material';
+import { Box, Portal, Stack } from '@mui/material';
 import { ECharts as EChartsInstance } from 'echarts/core';
 import React, { useState } from 'react';
 import useResizeObserver from 'use-resize-observer';
 import { EChartsDataFormat, UnitOptions } from '../model';
 import { TooltipContent } from './TooltipContent';
+import { TooltipHeader } from './TooltipHeader';
 import { getNearbySeriesData } from './nearby-series';
 import {
   CursorCoordinates,
   FALLBACK_CHART_WIDTH,
+  TOOLTIP_BG_COLOR_FALLBACK,
   TOOLTIP_MAX_HEIGHT,
-  TOOLTIP_MIN_WIDTH,
   TOOLTIP_MAX_WIDTH,
+  TOOLTIP_MIN_WIDTH,
   useMousePosition,
 } from './tooltip-model';
 import { assembleTransform } from './utils';
 
-interface TimeSeriesTooltipProps {
+export interface TimeSeriesTooltipProps {
   chartRef: React.MutableRefObject<EChartsInstance | undefined>;
   chartData: EChartsDataFormat;
-  isTooltipPinned: boolean;
   wrapLabels?: boolean;
   unit?: UnitOptions;
   onUnpinClick?: () => void;
+  pinnedPos: CursorCoordinates | null;
```

Comment:
suggestion: `pinnedCoordinates` (your call, this is totally readable as is)

Reply:
Oh I see you have `mousePos` below, that makes `pinnedPos` consistent with `mousePos`